README.md

```diff
@@ -297,6 +298,66 @@ When building with CMake, submodules are also recursively cloned automatically.
 can be passed as a CMake flag to disable this functionality. This is useful when you'd like to build CMake while using a
 different commit from a submodule.
 
+### Installation
+
+The C SDK libraries and platform abstractions can be installed to a file system
+through CMake.
+```sh
+cmake .. -DBUILD_DEMOS=0 -DBUILD_TESTS=0
+make install
+```
+Note that because `make install` will automatically build the `all` target, it may
+be useful to disable building demos and tests with `-DBUILD_DEMOS=0 -DBUILD_TESTS=0`
+unless they have already been configured.
+
+By default, the location of headers will be in the default system include path
+of the OS such as `/usr/local/include`. For libraries, this will likely
+be `/usr/local/lib`.
+
+Upon entering `make install`, the location of each library will be specified first
+followed by the location of all installed headers:
+```
+-- Installing: /usr/local/lib/libaws_iot_defender.so
```

Comment:
Should we support building and installing libraries as static archives if user wants? If that increases scope of this PR, it can be done in a follow-up PR.

Reply:
That is already a feature of CMake and can be specified with `-DBUILD_SHARED_LIBS=0`